src/credential/credentialController.ts

```diff
@@ -127,9 +128,9 @@ export class CredentialController {
     }
 
     private async signInWithAAD(): Promise<string> {
-        const authConfig = extensionConfig.auth[this.environmentController.env.toString()];
+        const authConfig = extensionConfig.auth[this.environmentController.env];
         const callbackUri = await vscode.env.asExternalUri(vscode.Uri.parse(`${vscode.env.uriScheme}://${EXTENSION_ID}/aad-authenticate`));
-        const signUrlTemplate = template.parse(`${AzureEnvironment.Azure.activeDirectoryEndpointUrl}/{tenantId}/oauth2/authorize` +
+        const signUrlTemplate = template.parse(`${AzureEnvironment.Azure.activeDirectoryEndpointUrl}{tenantId}/oauth2/authorize` +

```

Comment:
>{AzureEnvironment.Azure.activeDirectoryEndpointUrl} [](start = 49, length = 51)

To keep safe, we always use TrimEnd('/') and then add '/'. That would make the url always correct without external dependency

Reply:
+1

---
In reply to: [373908512](https://github.com/docascode/vscode-docs-build/pull/32#discussion_r373908512) [](ancestors = 373908512)